go.mod

```diff
@@ -3,13 +3,35 @@ module github.com/hashicorp/terraform-provider-tfe
 go 1.16
 
 require (
+	github.com/agext/levenshtein v1.2.3 // indirect
+	github.com/aws/aws-sdk-go v1.37.0 // indirect
+	github.com/fatih/color v1.12.0 // indirect
+	github.com/golang/protobuf v1.5.2 // indirect
 	github.com/google/go-querystring v1.1.0 // indirect
+	github.com/hashicorp/errwrap v1.1.0 // indirect
+	github.com/hashicorp/go-hclog v0.16.1 // indirect
+	github.com/hashicorp/go-multierror v1.1.1 // indirect
+	github.com/hashicorp/go-plugin v1.4.1 // indirect
 	github.com/hashicorp/go-retryablehttp v0.7.0 // indirect
 	github.com/hashicorp/go-slug v0.7.0
 	github.com/hashicorp/go-tfe v0.17.0
 	github.com/hashicorp/go-version v1.3.0
 	github.com/hashicorp/hcl v0.0.0-20180404174102-ef8a98b0bbce
-	github.com/hashicorp/terraform-plugin-sdk/v2 v2.7.0
-	github.com/hashicorp/terraform-svchost v0.0.0-20191011084731-65d371908596
+	github.com/hashicorp/hcl/v2 v2.10.0 // indirect
+	github.com/hashicorp/terraform-exec v0.14.0 // indirect
+	github.com/hashicorp/terraform-plugin-go v0.2.1
+	github.com/hashicorp/terraform-plugin-mux v0.1.1
+	github.com/hashicorp/terraform-plugin-sdk/v2 v2.5.0
```

Comment:
🤔 I was wondering the same; I thought for a moment it could be that the explicit import was removed from main.go and the plugin/mux packages use this version as a transitive dependency but...I don't think so?

Reply:
Good catches! Not sure how this happened, but updating this.